### ISSUE TITLE
openvino load tensorflow model.pd error

### ISSUE THREAD

**Issue by linjiaqin** [intel/analytics-zoo#965]
2019-10-16 01:11:10 INFO  InferenceSupportive$:45 - optimize tf image classification model execution time elapsed [10 s, 336 ms].
source_0	metal101.jpg	1571159470471	
2019-10-16 01:11:10 INFO  OpenVinoInferenceSupportive$:241 - tf image classification model optimized, log: 
stderr: + [ 8 -ne 8 ]
+ SAVED_MODEL_DIR=/home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel
+ INPUT_SHAPE=[1,224,224,3]
+ REVERSE_INPUT_CHANNELS=1
+ MEAN_VALUES=[123.68,116.78,103.94]
+ SCALE=1.0
+ INPUT=input
+ OPENVINO_IR_OUTPUT_DIR=/tmp/ZooVino4151219403393556622
+ MO_TF_FILE=/tmp/ZooVino6491586636955537423/model-optimizer/mo_tf.py
+ PARMS=--saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input
+ [ 1 -ne 0 ]
+ echo reverse_input_channels true
+ PARMS=--saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels
+ echo reverse_input_channels false
+ [ [123.68,116.78,103.94] != [] ]
+ PARMS=--saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels --mean_value [123.68,116.78,103.94] --scale 1.0
+ PARMS=--saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels --mean_value [123.68,116.78,103.94] --scale 1.0 --output_dir /tmp/ZooVino4151219403393556622
+ echo /tmp/ZooVino6491586636955537423/model-optimizer/mo_tf.py
+ echo --saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels --mean_value [123.68,116.78,103.94] --scale 1.0 --output_dir /tmp/ZooVino4151219403393556622
+ python3 /tmp/ZooVino6491586636955537423/model-optimizer/mo_tf.py --saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels --mean_value [123.68,116.78,103.94] --scale 1.0 --output_dir /tmp/ZooVino4151219403393556622
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted (core dumped)
 
stdout: reverse_input_channels true
reverse_input_channels false
/tmp/ZooVino6491586636955537423/model-optimizer/mo_tf.py
--saved_model_dir /home/linjiaqin/PycharmProjects/tianchitrash/package/savemodel --input_shape [1,224,224,3] --input input --reverse_input_channels --mean_value [123.68,116.78,103.94] --scale 1.0 --output_dir /tmp/ZooVino4151219403393556622
 
exitCode: 134

**this is the input of the model.pd**
MetaGraphDef with tag-set: 'serve' contains the following SignatureDefs:

signature_def['serving_default']:
  The given SavedModel SignatureDef contains the following input(s):
    inputs['input'] tensor_info:
        dtype: DT_FLOAT
        shape: (-1, 224, 224, 3)
        name: conv2d_1_input:0
  The given SavedModel SignatureDef contains the following output(s):
    outputs['output'] tensor_info:
        dtype: DT_FLOAT
        shape: (-1, 6)
        name: activation_4/Softmax:0
  Method name is: tensorflow/serving/predict
